src/main/resources/jsv-messages_en.properties

```diff
@@ -0,0 +1,70 @@
+$ref = {0}: has an error with ''refs''
```

Comment:
The 0 byte `jsv-messages_en.properties` should already fallback on `jsv-messages.properties`. Why is this needed?

Reply:
Okay thats right